RecoLocalTracker/SiStripZeroSuppression/plugins/SiStripZeroSuppression.cc

```diff
@@ -61,9 +61,13 @@ SiStripZeroSuppression::SiStripZeroSuppression(edm::ParameterSet const& conf)
     throw cms::Exception("Invalid option") << "When producing data in the hybrid format, the APV restorer must be "
                                               "configured with APVInspectMode='HybridEmulation'";
 
+  if ((!hybridInputs.empty()) && produceRawDigis) {
+    edm::LogWarning("SiStripZeroSuppression") << "Raw digis will not be saved for hybrid inputs";
```

Comment:
is it worth to throw here instead?
Given that this is a stream module, IIUC, we'll get nThread counts of these warnings in the log file.
If instead of exception this condition is OK and this message is only useful for experts, perhaps a LogInfo is better?

Reply:
It's OK as long as the product isn't used (and if it is, there will be a ProductNotFound exception), so I made it a LogInfo indeed (in any case this option would only be used by experts).